### PR TITLE
API-5344: upgrade play libraries

### PR DESCRIPTION
**Pull request by mattclark-zerogravit**

### CODE REVIEW

File: project/AppDependencies.scala [hunk 1]
```diff
@@ -8,18 +8,19 @@ object AppDependencies {
   lazy val jsoupVersion = "1.12.1"
   lazy val scalaCheckVersion = "1.14.0"
   lazy val enumeratumVersion = "1.6.2"
+  lazy val bootstrapVersion = "5.24.0"
```
Comment (worthydolt) [inline project/AppDependencies.scala:11]: 5.25.0 is available now

File: project/AppDependencies.scala [hunk 2]
```diff
@@ -39,12 +40,15 @@ object AppDependencies {
...
   lazy val testDependencies: Seq[ModuleID] = Seq(
-    "uk.gov.hmrc"                 %% "bootstrap-test-play-28"             % "5.16.0",
-    "org.jsoup"                   %  "jsoup"                              % jsoupVersion,
-    "com.github.tomakehurst"      %  "wiremock-jre8-standalone"           % "2.27.2",
-    "org.seleniumhq.selenium"     %  "selenium-java"                      % "2.53.1",
-    "org.seleniumhq.selenium"     %  "selenium-htmlunit-driver"           % "2.52.0",
-    "org.mockito"                 %% "mockito-scala-scalatest"            % "1.16.42",
-    "org.scalacheck"              %% "scalacheck"                         % scalaCheckVersion
+    "uk.gov.hmrc"                 %% "bootstrap-test-play-28"             % bootstrapVersion,
+    // "org.scalatestplus.play"  %%  "scalatestplus-play"        % "3.1.3",
```
Comment (worthydolt) [inline project/AppDependencies.scala:44]: It would be nice to remove commented lines

File: acceptance/common/BaseSpec.scala [hunk 3]
```diff
@@ -20,19 +20,18 @@ import com.github.tomakehurst.wiremock.WireMockServer
...
+import org.scalatest.matchers.should.Matchers
+import org.scalatestplus.play.guice.GuiceOneServerPerTest
+import play.api.inject.guice.GuiceApplicationBuilder
 import play.api.test.RunningServer
+import play.api.{Application, Mode}
 
 trait BaseSpec extends AnyFeatureSpec with BeforeAndAfterAll with BeforeAndAfterEach with Matchers with NavigationSugar with GuiceOneServerPerTest {
 
-  val stubPort = 11111
+//  val stubPort = 11111
```
Comment (worthydolt) [inline acceptance/common/BaseSpec.scala:33]: Does this need to stay?

**Review by worthydolt — APPROVED** [review]

File: project/AppDependencies.scala (hunk 2 above)
Comment (mattclark-zerogravit) [reply project/AppDependencies.scala]: done

File: project/AppDependencies.scala (hunk 1 above)
Comment (mattclark-zerogravit) [reply project/AppDependencies.scala]: Ok will update to that version

**Pull request merged**